devtools/server/actors/replay/connection.js

```diff
@@ -1189,6 +1195,79 @@ async function uploadAllSourcemapAssets({
   ]);
 }
 
+async function writeAllSourcemapAssets({
+  recordingId,
+  sourceMapURL,
+  sourceMapBaseURL,
+  targetContentHash,
+  targetURLHash,
+  targetMapURLHash,
+}, recording) {
```

Comment:
Why pass in the recording instead of the `browser` like `uploadAllSourcemapAssets` does?

Reply:
That was a remnant of #760